### ISSUE TITLE
Support for SELECT JSON queries

### ISSUE THREAD

**Issue by richbolen** [outworkers/phantom#500]
Can you add support for "SELECT JSON FROM" type queries.  We use this heavily.  We're currently using a modified (very old) version of your driver and we would love to use your official version but lack of this feature is preventing us from doing so.

**Comment by alexflav23** [comment]
Hi @richbolen,

There is a variation of this going live together with a much larger changeset, including all the things we've discussed such as case sensitive table names and using ignore nulls. This is still very actively worked on, it just required a very large changeset to do properly, especially around propagating table serialization configuration globally.

We will keep you well up to date, and this is well on track.

Regards.

**Comment by alexflav23** [comment]
Hi @richbolen,

Apologies for the delay in delivering this, time has not been kind recently. Regardless, this will now be available as of phantom 1.28.5. The syntax looks like this:

```scala
def getJson(id: UUID): Future[Option[String]] = {
  database.table.select.json().where(_.id eqs id).one()
}
```

`.json()` will make select queries return a `String`, as phantom cannot make any assumptions about which JSON library end users want to use, but you can quite easily `.map(Json.parse(_)` or something similar.

**Comment by richbolen** [comment]
Apologies for the delay on responding to this.  Thanks very much.  I hope to be looking at upgrading Cassandra to the latest version soon (and the latest version of your driver).
